zookeeper-server/src/main/java/org/apache/zookeeper/server/auth/znode/groupacl/ZNodeGroupAclProperties.java

```diff
@@ -65,12 +70,17 @@ public static ZNodeGroupAclProperties getInstance() {
   // Meaning the znode will have (world:anyone, r) ACL
   private static final String OPEN_READ_ACCESS_PATH_PREFIX =
       ZNODE_GROUP_ACL_CONFIG_PREFIX + "openReadAccessPathPrefix";
+  // If the server is dedicated for one namespace, use this config property to define the namespace,
+  // and enable connection filtering feature for this namespace
+  public static final String NAMESPACE = ZNODE_GROUP_ACL_CONFIG_PREFIX + "namespace";
```

Comment:
nit: we should find a different term. something like dedicated domain probably is more clear?